documentation/tutorial.md

```diff
@@ -9,6 +10,7 @@ Before we start be sure you have the following requirements:
 - At least 20GB of available storage.
 - Access to the internet for downloading the required snaps and charms.
 
+
```

Comment:
I think it would be nice to include a summary of what this covers, either as a bulletpoint list again, or by adding more to the intro. For example, that it includes setting up LXD and Juju, that it includes how to scale up and down, use TLS etc. It is more than 'getting it up and running'